modules/codegen-plugin/src/smithy4s/codegen/JsonConverters.scala

```diff
@@ -61,13 +61,17 @@ private[smithy4s] object JsonConverters {
     )
 
   // format: off
-  type GenTarget = List[os.Path] :*: os.Path :*: os.Path :*: Set[FileType] :*: Boolean:*: Option[Set[String]] :*: Option[Set[String]] :*: List[String] :*: List[String] :*: List[String] :*: List[os.Path] :*: LNil
+  type GenTarget = List[os.Path] :*: String :*: String :*: Set[FileType] :*: Boolean:*: Option[Set[String]] :*: Option[Set[String]] :*: List[String] :*: List[String] :*: List[String] :*: List[os.Path] :*: LNil
```

Comment:
**suggestion:** I would rather restore `os.Path` here, and make `os.Path`'s format based on the toString. For cases where we do want to use `os.Path` as a checksum, IMO let's newtype it - that way we'll get closer to Mill's `PathRef`, which is the inspiration for our checksumming of paths.

Most importantly, this would save us the trouble of rendering and re-parsing the path. "stringly typed" isn't great

Reply:
Agreed

Reply:
yeah